assets/xml/objects/object_dkjail_obj.xml

```diff
@@ -1,9 +1,9 @@
 ﻿<Root>
     <File Name="object_dkjail_obj" Segment="6">
```

Comment:
Can you put a comment up here explaining what this object contains? Looks like it was missed.